src/core/credentials/transport/tls/spiffe_utils.h

```diff
@@ -28,6 +28,8 @@
 
 namespace grpc_core {
 
+std::string BundleRootToPem(absl::string_view bundle_root);
```

Comment:
Since this function is general purpose and not specific to SPIFFE, should it be in ssl_transport_security_utils.h or ssl_utils.h instead?

Reply:
Agreed, given your reply let's leave it here.